src/goto-analyzer/static_verifier.h

```diff
@@ -38,9 +38,13 @@ void static_verifier(
   const ai_baset &ai,
   propertiest &properties);
 
-// clang-format off
-enum class ai_verifier_statust { TRUE, FALSE, BOTTOM, UNKNOWN };
-// clang-format on
+enum class ai_verifier_statust
+{
+  TRUE,
+  FALSE_IF_REACHABLE,
+  NOT_REACHABLE,
+  UNKNOWN
+};
```

Comment:
Perhaps don't do the `clang-format off` in the earlier commit?

Reply:
Done.